Assets/VRTK/Scripts/Controls/3D/VRTK_Lever.cs

```diff
@@ -30,7 +30,7 @@ public enum LeverDirection
 
         private Rigidbody rb;
         private VRTK_InteractableObject io;
-        private HingeJoint hj;
+        protected HingeJoint hj;
```

Comment:
move the protected variables above the private variables, so the order should be

`public` then `protected` then `private`